protoc-gen-gogo/grpc/grpc.go

```diff
@@ -129,11 +129,15 @@ func (g *grpc) GenerateImports(file *generator.FileDescriptor) {
 	if len(file.FileDescriptorProto.Service) == 0 {
 		return
 	}
-	g.P("import (")
-	g.P(contextPkg, " ", strconv.Quote(path.Join(g.gen.ImportPrefix, contextPkgPath)))
-	g.P(grpcPkg, " ", strconv.Quote(path.Join(g.gen.ImportPrefix, grpcPkgPath)))
-	g.P(")")
-	g.P()
+	grpcImports := []string{
+		path.Join(g.gen.ImportPrefix, contextPkgPath),
```

Comment:
this bit is incorrect - there is no need for this `path.Join` - the `imports` object already takes care of the import prefix. In CockroachDB, this causes the import prefix to appear twice.